src/exercises/exerciseTypes/NextNavigation.js

```diff
@@ -45,7 +45,7 @@ export default function NextNavigation({
       {isCorrect && userIsCorrect && (
         <div className="next-nav-feedback">
           <img
-            src={"/static/icons/zeeguu-icon-correct.png"}
+            src={"https://zeeguu.org/static/icons/zeeguu-icon-correct.png"}
```

Comment:
![image](https://github.com/zeeguu/web/assets/17390076/e0a46277-4f66-42bc-9298-6a4d9d024f47)

We use this in elements that the extension is using so they are correctly displayed. Maybe we could make a wrapper function that based on where it's called it fills it correctly.

Reply:
Thanks for the explanation. Yes. A wrapper function would allow us to change everything in one place should it ever be needed, and also allow us to add a comment to explain it's existence :)